### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by pjhampton**
In an attempt to make Barrel approachable and more user friendly without altering the code, I have updated the read me. If any **features** or **goals** are missing, please leave a comment on the code :smile:

https://github.com/pjhampton/barrel-platform/tree/readme_overhaul

Barrel Readme redesign

A couple of things worth thinking about.

- Homebrew install 
- remove port 80 description
- docker image

cc/ @benoitc @sivsushruth

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,28 +93,10 @@ On FreeBSD all ports can be made accessible to all users by issuing:
     $ sysctl net.inet.ip.portrange.reservedhigh=0
 
 
-## Ownership and License
```
Comment (benoitc) [inline README.md]: hrm shouldn't we keep it?

**Comment by benoitc** [comment]
I really like these changes. Icommented about the c4.1 removal. we should imo keep it, if not we need some contributions guidelines. thoughts?

**Comment by pjhampton** [comment]
@benoitc my plan was to move it to a separate file **/CONTRIBUTING.md** in the root directory to make full use of github - https://github.com/blog/1184-contributing-guidelines. As we are only hosting the source code and accepting PRs on github, this is probably the best approach. 

What do you think?

**Comment by pjhampton** [comment]
I have now removed the Installation and Port 80 stuff and just linked it the documentation :smile: I think a visitor should be able to read the entire readme in 60 seconds or less.

I currently have two problems:

- **(such as offline or in other conditions)**? What does this mean? There is an important word missing between *other* and *conditions*.

- *Get changes and replicate from a view index* - What is this? Is this worded properly?

**Comment by benoitc** [comment]
@pjhampton Sorry for the late answer.  Awesome work, I really like it, thanks a lot :) One thing though, the code of conduct part should be removed, everything about contributing is already covered by the C4.1 and this what count.

Responses to other points follow:

1) I meant "such as being offline, disconnected and  similar contexts" . Is that more clear?
2) It's related to the [view changes](https://github.com/rcouch/rcouch/wiki/View-Changes) and the [replication using a view change](https://github.com/rcouch/rcouch/wiki/Replication-with-view-changes). I plane to write the documentation this afternoon.
3) I meant mobile for phones and tablets.


I'm available online if you want to have a quick chat :)

**Comment by pjhampton** [comment]
@benoitc updated :smile: I'll jump on irc

**Comment by benoitc** [comment]
@pjhampton thanks! merged

**Pull request merged**